src/App.vue

```diff
@@ -1,13 +1,72 @@
 <template>
-  <h4>Hello World</h4>
+  <div class="screen">
+    <router-view></router-view>
+    <div
+      v-if="this.$route.name == 'Challenge'"
+      style="width: 100%; min-height: 81px; height: 81px"
+    ></div>
+    <BottomNav v-if="showBottomNav" :bottomMenu="bottomMenu"></BottomNav>
+  </div>
 </template>
 
 <script>
-export default {
-  data() {
-    return {};
-  },
-};
+  import BottomNav from "@/components/BottomNav.vue";
+
+  export default {
+    components: {
+      BottomNav,
+    },
+    data() {
+      return {
+        bottomMenu: ["write", "calendar", "challenge"],
```

Comment:
Move to store

```js
// type 1
const BOTTOM_MENU = ["write", "calendar", "challenge"]

// type 2
const BottomMenu = {
     write: "wirte",
     calendar: "calendar",
    chanllenge: "chanllenge"
}

// usage
{{ BottomMenu.write }}
```

Reply:
Solved.
move to state() in sotre.js and
import {mapState} from "vuex"; in BottomNav.vue (Component)